utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -135,6 +135,21 @@ function Invoke-ResourceRemoval {
             }
             break
         }
+        'Microsoft.DevTestLab/labs' {
+            $resourceGroupName = $resourceId.Split('/')[4]
+            $resourceName = Split-Path $resourceId -Leaf
+            if ($PSCmdlet.ShouldProcess("DevTestLab Lab [$resourceName]", 'Remove')) {
+                Get-AzResourceLock -ResourceGroupName $resourceGroupName |
```

Comment:
Is that a special lock? 

The resource locks should already automatically be the first to be removed as per our removal logic (with a few seconds propagation).

Reply:
Yes, only for locks we enable. It leaves these locks behind as they're special to DTL